jerry-core/parser/regexp/re-compiler.h

```diff
@@ -57,8 +57,14 @@
 #define RE_OP_CHAR_CLASS                                    26
 #define RE_OP_INV_CHAR_CLASS                                27
```

Comment:
@LaszloLango, I think that it is better to use `enum`. If we would switch to C++, it would be even better to use `enum class`.